custom_components/hwam_stove/translations/de.json

```diff
@@ -0,0 +1,202 @@
+        "name": "Ventile"
+      },
+      "safety_alarms_bad_configuration": {
+        "name": "Konfiguration"
+      },
+      "safety_alarms_valve_disconnect": {
+        "name": "Ventil trennen"
+      },
+      "safety_alarms_valve_calibration_error": {
+        "name": "Ventile kalibriert"
+      },
+      "safety_alarms_stove_overheat": {
+        "name": "Überhitzung"
+      },
+      "safety_alarms_door_open_too_long": {
+        "name": "Tür zu lange auf",
+        "state": {
+          "off": "Nein",
+          "on": "Ja"
+        }
+      },
+      "safety_alarms_manual_safety_alarm": {
+        "name": "Manualle Sicherheit"
+      },
+      "safety_alarms_stove_sensor_fault": {
+        "name": "Sensoren"
+      }
+    },
+    "button": {
+      "start": {
+        "name": "Start"
+      },
+      "sync_clock": {
+        "name": "Synkronisiere Uhr"
+      }
+    },
+    "datetime": {
+      "date_and_time": {
+        "name": "Datum und Zeit"
+      }
+    },
+    "number": {
+      "burn_level": {
+        "name": "Brenngrad soll"
+      }
+    },
+    "sensor": {
+      "algorithm": {
+        "name": "Algorithmus"
+      },
+      "message_id": {
+        "name": "Message ID"
+      },
+      "new_firewood_estimate": {
+        "name": "Feuerholz nachlegen ~"
+      },
+      "night_lowering": {
+        "name": "Nachtabsenkung",
+        "state": {
+          "disabled": "Deaktiviert",
+          "init": "Initialisierung",
+          "on_day": "Tag",
+          "on_night": "Nacht",
+          "on_manual_night": "Manuell Nacht"
+        }
+      },
+      "operation_mode": {
+        "name": "Arbeitsmodus",
+        "state": {
+          "init": "Initialisierung",
+          "self_test": "Selbsttest",
+          "normal": "Normal",
+          "temperature_fault": "Temperatur Fehler",
+          "o2_fault": "Sauerstoff Fehler",
+          "calibration": "Kalibartion",
+          "safety": "Sicherheit",
+          "manual": "Manuell",
+          "motor_test": "Motor Test",
+          "slow_combustion": "Langsame Verbrennung",
+          "low_voltage": "Niedrige Spannung (Volt)"
+        }
+      },
+      "oxygen_level": {
+        "name": "Sauerstoff Level"
+      },
+      "phase": {
+        "name": "Brennphase",
+        "state": {
+          "ignition": "Anzünden",
+          "burn": "Brennen",
+          "glow": "Glut halten",
+          "standby": "Standby"
+        }
+      },
+      "room_temperature": {
+        "name": "Raumtemperatur"
+      },
+      "stove_temperature": {
+        "name": "Rauchgastemperatur"
+      },
+      "time_since_remote_message": {
+        "name": "Zeit seit remote Nachricht"
+      },
+      "time_to_new_firewood": {
+        "name": "Zeit für neues Feuerholz"
+      },
+      "valve_1_position": {
+        "name": "Klappe Primär"
+      },
+      "valve_2_position": {
+        "name": "Klappe Sekundär"
+      },
+      "valve_3_position": {
+        "name": "Klappe Tertiär"

```

Comment:
We used `Ventile` for `Valve` earlier on in this file. Is there a reason we can't stick to a single term for consistency?

Reply:
Following the manufacturer is a good place to start, but not mandatory. If we can do better, we should.
If `Klappe` is a word you would commonly use to describe the valves in the stove then stick with that, if not then use something else. Just make it consistent across the translation.

Like I mentioned above, most terminology in the English version comes from their older Android app, but I had to improve some terms there as well.